src/selectbox_abstract.coffee

```diff
@@ -108,6 +108,8 @@ class SelectBoxAbstract
         # Replicate changes from the original select box to the fake one
         @orig.on 'reform.sync change DOMSubtreeModified', @refresh
         
+        @body.on 'reform.close', => @close()
```

Comment:
Why do you want to close all reforms at once? I think you should have this event on `@orig` element. A reform instance should not care about other instances.